formats/gltf/model.go

```diff
@@ -51,3 +51,86 @@ type PolyformTexture struct {
 	URI     string
 	Sampler *Sampler
 }
+
+func (pm PolyformMaterial) equal(other PolyformMaterial) bool {
```

Comment:
We need to check normal texture, and extras too

Reply:
Hope it's everything now.